### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by U2GT** [boinx/mimoLive-Stream-Deck-Plugin#4]
Hello, I'm new to Mimo so this may already be possible with the Streamdeck plugin.  If not, can it be added?

I can toggle between cameras in one Placer with variants, and I can also turn cameras on live if each camera has it's own placer.  But how do I set it up so that I can have one button for each camera on the Streamdeck and be able to choose which camera I want to switch to while I'm live?  If each camera has it's own placer, I can turn it on live, but then I have to turn it off before switching to another camera.  If I use one placer with several camera variants, it will toggle between all the cameras live, but I want to go to one particular camera while live and don't want to toggle through the others.  

In other words, I would like to have a command in the Streamdeck dropdown list that selects the camera variant, just like when I press the "live" button next to the variant with my mouse pointer to choose which camera to go to.  I prefer to not have to grab my mouse and go to the Mimo software to switch cameras.  I would like to just press "Cam 1", "Cam 2" etc on my Streamdeck while live and move to the camera I wish to use (without turning the previous one off first).

**Comment by oliverbreidenbach** [comment]
Hi @U2GT Thanks for the feature request. A "swicher" is typically implemented in mimoLive using the "Video Switcher" layer. You can use the API Endpoints of the Buttons that switch to the desired source in the Stream Deck app and make buttons that switch to a specific camera source.

**Comment by U2GT** [comment]
Thanks.  I just asked Art on the FB forum for more info.  Having trouble with what button to choose and how to set it up.  Is there a detailed manual on this subject?  I'm pretty tech savvy, but the detailed button set up is not intuitive like Ecamm.  It's hard to tell what I'm supposed to do.  Art said I copy the URL and I see that, but I don't know where to paste it and also don't know what else to set up.  Thank you!
